js/repeater.js

```diff
@@ -699,6 +699,13 @@
 			var height;
 			var viewportMargins;
 
+			var $hiddenParents = this.$element.parents().addBack().not(':visible');
```

Comment:
change to `$hiddenParents = this.$element.parents(':hidden').addBack();`
easier to read